src/axom/multimat/multimat.cpp

```diff
@@ -701,15 +877,32 @@ void MultiMat::convertLayoutToSparse()
 {
   for(unsigned int map_i = 0; map_i < m_fieldMappingVec.size(); map_i++)
   {
-    convertFieldToSparse(map_i);
+    try
+    {
+      convertFieldToSparse(map_i);
+    }
+    catch(const std::runtime_error& ex)
+    {
+      SLIC_WARNING("Multimat: cannot convert field \""
+                   << m_fieldNameVec[map_i]
+                   << "\" to sparse layout, skipping...");
+    }
```

Comment:
Should we be introducing exceptions into the conversion operations? Are there downsides of doing this?